packages/labs/src/components/popover/popover2.tsx

```diff
@@ -85,7 +93,7 @@ export interface IPopover2Props extends IOverlayableProps, IProps {
      * When modal popovers are opened, they become focused.
      * @default false
      */
-    isModal?: boolean;
+    hasBackdrop?: boolean;
```

Comment:
Does the description need to change if we're changing the prop name? (e.g. is it worth using the word "modal" anymore?)